### DISCUSSION TITLE
v1.0-alpha2

### DISCUSSION THREAD

**Discussion by bitbrain** [bitbrain/pandora#90] [Announcements]
The 2nd alpha release of Pandora has arrived! While [Version 1.0](https://github.com/bitbrain/pandora/milestone/1) still seems lightyears away, contributors have been hard at work addressing issues and resolving bugs.

## What's Changed
* :book: Minor typo in sample code and error message by @russmatney in https://github.com/bitbrain/pandora/pull/75
* :sparkles: Tooltips & minor whitespace tweak by @tylercchase in https://github.com/bitbrain/pandora/pull/76
* :sparkles: Added string representation for model resources. by @VojtaStruhar in https://github.com/bitbrain/pandora/pull/78
* :bug: Setting a reference property by @VojtaStruhar in https://github.com/bitbrain/pandora/pull/81
* :bug: Fix typo on editor interface page by @eth0net in https://github.com/bitbrain/pandora/pull/86
* :bug: Fix invalid entity instantiation by @bitbrain in https://github.com/bitbrain/pandora/pull/74
* :bug: Fallback to entity instantiation when script is invalid by @bitbrain in https://github.com/bitbrain/pandora/pull/87
* :bug: Do not use custom constructor for resource by @bitbrain in https://github.com/bitbrain/pandora/pull/88

## New Contributors
* @russmatney made their first contribution in https://github.com/bitbrain/pandora/pull/75
* @tylercchase made their first contribution in https://github.com/bitbrain/pandora/pull/76
* @VojtaStruhar made their first contribution in https://github.com/bitbrain/pandora/pull/78
* @eth0net made their first contribution in https://github.com/bitbrain/pandora/pull/86

**Full Changelog**: https://github.com/bitbrain/pandora/compare/v1.0-alpha1...v1.0-alpha2

<hr /><em>This discussion was created from the release <a href='https://github.com/bitbrain/pandora/releases/tag/v1.0-alpha2'>v1.0-alpha2</a>.</em>

**Comment by VojtaStruhar** [comment]
Seeing my @ mentined here really does bring me happiness 😄 I hope to get more involved in the future, I really like where this plugin is going!!

Let's go guys!
